mkdocs.yml

```diff
@@ -174,6 +174,8 @@ plugins:
           merge_init_into_class: true
           extensions:
             - docs/plugins/griffe_doclinks.py
+        import:
+          - https://docs.python.org/3.13/objects.inv
```

Comment:
Let's use 3.12

Reply:
It's also possible to use `https://docs.python.org/3/objects.inv`, which IIUC points to latest stable release (3.12 currently).

Reply:
Yes the reason I wanted to use 3.13 was in prevision of the documentation of deprecated fields, which can make use of `warnings.deprecated`. However I don't think it will be necessary for now to have the cross ref for this